src/test/java/org/assertj/core/description/JoinDescription_value_Test.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2019 the original author or authors.
+ */
+package org.assertj.core.description;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.util.Lists.list;
+
+import org.assertj.core.internal.TestDescription;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Tests for {@link JoinDescription#value()}.
+ *
+ * @author Edgar Asatryan
+ */
+class JoinDescription_value_Test {
```

Comment:
Minor: add GIVEN WHEN THEN comment for test readability

Reply:
also add `@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)`